wd_core.au3

```diff
@@ -250,21 +250,19 @@ EndFunc   ;==>_WD_CreateSession
 ; Return values .: Success - 1
 ;                  Failure - 0 and sets @error to $_WD_ERROR_Exception
 ; Author ........: Danp2
-; Modified ......:
+; Modified ......: mLipok
```

Comment:
This is only remark :
so as not to forget to look at `wd_helper.au3` and also check all `SetError(__WD_Error(......` usage.
But firstly we should to focus on changes in `wd_core.au3`

Reply:
No longer needed.